src/GitVersionCore/StringFormatWith.cs

```diff
@@ -27,16 +27,44 @@ public static string FormatWith<T>(this string template, T source)
 
             // {MajorMinorPatch}+{Branch}
             var objType = source.GetType();
+            const string ENV_VAR_INDICATOR = "env:";
```

Comment:
Perhaps this `const` can be moved up alongside the `TokensRegex`?

Reply:
Addressed in the latest commit.